packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterPanel.tsx

```diff
@@ -43,16 +72,12 @@ export function GridFilterPanel() {
     [apiRef],
   );
 
-  React.useEffect(() => {
-    if (filterModel.items.length === 0) {
-      addNewFilter();
-    }
-  }, [addNewFilter, filterModel.items.length]);
+  const hasMultipleFilters = items.length > 1;
```

Comment:
Can we organize it a bit better - maybe put this line after the `items` useMemo since this is on what it depends. The others are useCallbacks or just normal arrow functions.